src/Login/Cookie/CookieLogin.php

```diff
@@ -63,7 +63,7 @@ public function addCookie(CookieLoginIdentityInterface $identity, ResponseInterf
         ], JSON_THROW_ON_ERROR | JSON_UNESCAPED_SLASHES | JSON_UNESCAPED_UNICODE);
 
         return (new Cookie($this->cookieName, $data))
-            ->withMaxAge($identity->getCookieLoginDuration() ?? $this->duration)
```

Comment:
Potentially that allows setting cookie login duration per user but I'm not sure it's actually needed so likely it's OK to remove it.